packages/contractkit/src/wrappers/VestingInstanceWrapper.ts

```diff
@@ -0,0 +1,308 @@
+   */
+  isPaused: () => Promise<boolean> = proxyCall(this.contract.methods.paused)
+
+  /**
+   * Indicates if the vesting is revocable or not
+   * @return A boolean indicating revocable vesting (true) or none-revocable(false).
+   */
+  isRevokable: () => Promise<boolean> = proxyCall(this.contract.methods.revocable)
+
+  /**
+   * Indicates if the vesting is revoked or not
+   * @return A boolean indicating revoked vesting (true) or none-revoked(false).
+   */
+  isRevoked: () => Promise<boolean> = proxyCall(this.contract.methods.revoked)
+
+  /**
+   * Returns the time at which the vesting was revoked
+   * @return The timestamp of the vesting revokation
+   */
+  getRevokeTime: () => Promise<string> = proxyCall(this.contract.methods.revokeTime)
+
+  /**
+   * Returns the end time of the set pause in UNIX
+   * @return The end time of the set pause in UNIX
+   */
+  getPauseEndTime: () => Promise<string> = proxyCall(this.contract.methods.pauseEndTime)
+
+  /**
+   * Returns the withdrawable amount at a specified timestamp.
+   * @param timestamp The timestamp in question.
+   * @return The amount which could be withdrawn at that timestamp.
+   */
+  getWithdrawableAmountAtTimestamp: (timestamp: string | number) => Promise<BigNumber> = proxyCall(
+    this.contract.methods.getWithdrawableAmountAtTimestamp,
+    undefined,
+    toBigNumber
+  )
+
+  /**
+   * Pause a vesting instance
+   * @param pausePeriod The duration of the pause period in seconds
+   * @return A CeloTransactionObject
+   */
+  async pauseVesting(pausePeriod: string | number): Promise<CeloTransactionObject<void>> {
+    return toTransactionObject(this.kit, this.contract.methods.pause(pausePeriod))
+  }
+
+  /**
+   * Revoke a vesting instance
+   * @param pausePeriod The timestamp at which the revoking is to take place
+   * @return A CeloTransactionObject
+   */
+  async revokeVesting(revokeTimestamp: string | number): Promise<CeloTransactionObject<void>> {
+    return toTransactionObject(this.kit, this.contract.methods.revoke(revokeTimestamp))
+  }
+
+  /**
+   * Unlocks gold that becomes withdrawable after the unlocking period.
+   * @param value The amount of gold to unlock.
+   */
+  lockGold: (value: NumberLike) => CeloTransactionObject<void> = proxySend(
+    this.kit,
+    this.contract.methods.lockGold,
+    tupleParser(parseNumber)
+  )
+
+  /**
+   * Unlocks gold that becomes withdrawable after the unlocking period.
+   * @param value The amount of gold to unlock.
+   */
+  unlockGold: (value: NumberLike) => CeloTransactionObject<void> = proxySend(
+    this.kit,
+    this.contract.methods.unlockGold,
+    tupleParser(parseNumber)
+  )
+
+  /**
+   * Relocks gold that has been unlocked but not withdrawn.
+   * @param index The index of the pending withdrawal to relock.
+   * @param value The value to relock.
+   */
+  relockGold: (index: number, value: string | number) => CeloTransactionObject<void> = proxySend(
```

Comment:
Take a look at LockedGold.relock, we'll probably want to do something similar:

https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/src/wrappers/LockedGold.ts#L82

Reply:
adopted